src/go/k8s/tests/e2e/update-node-port/01-new-image.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: up-node-ports
+spec:
+  version: "latest"
```

Comment:
just wondering - can't we just assert the ports in the update-image test to not add new test and make the whole run slower?

Reply:
this question still stands :)

Reply:
I just did it. The src/go/k8s/tests/e2e/update-image/00-assert.yaml file changed to src/go/k8s/tests/e2e/update-image-and-node-port/00-assert.yaml and all other files accordingly.